src/test/unittest/ut_pmem2_config.c

```diff
@@ -46,3 +46,21 @@ ut_pmem2_config_delete(const char *file, int line, const char *func,
 
 	UT_ASSERTeq(*cfg, NULL);
 }
+
+/*
+ * prepare_config -- fill pmem2_config in minimal scope
+ */
+void
+ut_prepare_config(const char *file, int line, const char *func, 
```

Comment:
this should also probably be in the `ut_setup.h`

Reply:
It is difficult to do. Ut_setup.c requires a few c files from core and libpmem2. In ut_pmem2_config.c are functions using only API functions. If I will move ut_prepare_config() to ut_setup.c all test using this function, will require changes in Makefiles

Reply:
This all stems from the fact that we use internal symbols in almost every tests, even when that serves no purpose. That's the original problem that I think should be addressed.

Reply:
In my opinion, it separates unit tests from integration tests.